### PR TITLE
Feature checksum unit test

### PR DESCRIPTION
**Pull request by Raymundogs**

### CODE REVIEW

**Review by SaulCM — APPROVED** [review]
Seems ok.

File: src/test/java/org/fundacion/jala/converter/core/ChecksumServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+public class ChecksumServiceTest {
+
+    @Rule
+    public TemporaryFolder folder = new TemporaryFolder();
+    
+    @Test
+    public void getFileChecksumShouldReturnFileChecksum() throws IOException, ChecksumException {
```
Comment (Paola-Xaq) [inline src/test/java/org/fundacion/jala/converter/core/ChecksumServiceTest.java]: Should we test IOException? if is something we are not implementing.

File: src/test/java/org/fundacion/jala/converter/core/ChecksumServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        File tempFile = folder.newFile("myFile.txt");
+        String actual = getFileChecksum(tempFile.getPath());
+        String expected = "d41d8cd98f00b204e9800998ecf8427e";
+        assertEquals(expected, actual);
+    }
+
+    @Test(expected = NullPointerException.class)
+    public void getFileChecksumShouldThrowChecksumExceptionWithNullInput() throws ChecksumException {
```
Comment (Paola-Xaq) [inline src/test/java/org/fundacion/jala/converter/core/ChecksumServiceTest.java]: Test convention wasn't supposed to be.. all test name start with should? shouldGetChecksum....?

File: src/test/java/org/fundacion/jala/converter/core/ChecksumServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+    @Test
+    public void repeatedChecksumShouldReturnTrueWithCorrectChecksum() throws IOException, ChecksumException {
+        File tempFile = folder.newFile("myFile.txt");
+        Boolean actual = repeatedChecksum(tempFile.getPath(), "d41d8cd98f00b204e9800998ecf8427e");
+        assertTrue(actual);
+    }
+
+    @Test
+    public void repeatedChecksumShouldReturnFalseWithIncorrectChecksum() throws IOException, ChecksumException {
```
Comment (Paola-Xaq) [inline src/test/java/org/fundacion/jala/converter/core/ChecksumServiceTest.java]: I think only ChecksumException should be tested.

File: src/test/java/org/fundacion/jala/converter/core/VideoConverterTest.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        File thumbnailFile = new File(filePath + "/Perrito.png");
+        assertEquals(expected, actual);
+    }
+
+    @Test(expected = NullPointerException.class)
+    public void convertVideoShouldThrowConverterExceptionWithNullParameter() throws ConverterException {
+        VideoConverter videoConverter = new VideoConverter(null);
+        videoConverter.convertVideo();
+    }
+}
```
Comment (Paola-Xaq) [inline src/test/java/org/fundacion/jala/converter/core/VideoConverterTest.java]: Add a new extra line at the end.

File: src/test/java/org/fundacion/jala/converter/core/parameter/VideoParameterTest.java [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+    }
+
+    @Test
+    public void hasThumbnailShouldReturnThumbnail() {
+        VideoParameter videoParameter = new VideoParameter("resources/archive/Perrito.mp4", "mpeg",
+                "720", true, 30, 1280, 720, false);
+        boolean actual = videoParameter.hasThumbnail();
+        assertTrue(actual);
+    }
+}
```
Comment (Paola-Xaq) [inline src/test/java/org/fundacion/jala/converter/core/parameter/VideoParameterTest.java]: Add a new line at the end.

**Pull request merged**